### ISSUE TITLE
"Create lmms working directory?" prompt is unreachable

### ISSUE THREAD

**Issue by Wallacoloo** [LMMS/lmms#2232]
When a user starts lmms for the first time, they are supposed to be prompted to create the ~/lmms working directory (code [here](https://github.com/LMMS/lmms/blob/5592d07c80c0bf81acbaca9552fe5ce31e797b1e/src/core/ConfigManager.cpp#L429)).

However, one condition of creating that prompt is that the gui must exist. Yet `ConfigManager::loadConfigFile()` is only ever called from [main.cpp:399](https://github.com/LMMS/lmms/blob/5592d07c80c0bf81acbaca9552fe5ce31e797b1e/src/core/main.cpp#L399), at which point the gui has not been created. So this code is unreachable.

This can be verified simply by deleting or temporarily renaming ~/lmms to something else and then launching lmms. You will not be prompted to create the working directory.

**Comment by tresf** [comment]
Thanks... This explains why I keep getting blank artwork on my msys2 builds.

**Comment by Wallacoloo** [comment]
If this is to be fixed, I propose that the prompt functionality be moved *outside* of ConfigManager.cpp. That is, the code in main.cpp would look something like:

```c++
if (!ConfigManager::inst()->workingDirExists())
{
    // prompt to create working directory
}
ConfigManager::inst()->loadConfig();
```

This way we avoid creating a QMessageBox from within the core (i.e. we improve some decoupling between core & gui while we're at it).

**Comment by Wallacoloo** [comment]
Fixed via #2233

**Comment by Wallacoloo** [comment]
Fixed via #2238
